src/D2L.CodeStyle.TestAnalyzers/NUnit/CategoryAnalyzer.cs

```diff
@@ -69,9 +70,12 @@ internal sealed class CategoryAnalyzer : DiagnosticAnalyzer {
 					return;
 				}
 
+				// TODO: can this actually be null?
+				var syntaxReference = attribute.ApplicationSyntaxReference!;

```

Comment:
comment is just there as a hint that this `!` is a guess.
Clearly it hasn't caused troubles yet 🤷